weave

```diff
@@ -663,6 +663,17 @@ http_call_ip() {
     curl --connect-timeout 3 -s -S -X $http_verb "$@" http://$ip:$port$url
 }
 
+http_call_unix() {
+    container="$1"
+    socket="$2"
+    http_verb="$3"
+    url="$4"
+    shift 4
+    output=$(docker exec $container curl -s -S -X $http_verb --unix-socket $socket "$@" http:$url)
+    [[ -n "$output" ]] || return 1
+    echo $output
```

Comment:
Not sure. It's not mentioned in their changelog. Might have been with the move to libcontainer.